sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/analysis/unresolved.scala

```diff
@@ -141,6 +141,10 @@ case class UnresolvedFunction(
   override def nullable: Boolean = throw new UnresolvedException(this, "nullable")
   override lazy val resolved = false
 
+  override def prettyString: String = {
+    s"${name}(${children.map(_.prettyString).mkString(",")})"
```

Comment:
@rxin @marmbrus It is better to change the `prettyString` of `UnresolvedFunction` to get rid of that `'` (e.g. `'abs`). The impact of this change is that if any user rely on that `'`, his/her code will break.